apps/showcase/package.json

```diff
@@ -78,7 +79,7 @@
     "highlight.js": "^11.8.0",
     "intl-messageformat": "~10.7.0",
     "marked": "^12.0.0",
-    "monaco-editor": "~0.52.0",
+    "monaco-editor": "~0.50.0",
```

Comment:
is that still needed? what issue do we have we `>0.50.0` ?

Reply:
I just tested and there is still a problem with version `>0.50.0` of `monaco-editor`, it seems that the linked issues in their repository are not yet resolved either...